src/components/molecules/VenuePartygoers/VenuePartygoers.tsx

```diff
@@ -15,7 +15,7 @@ import "./VenuePartygoers.scss";
 
 const filterVenuePartygoers = (partygoers: User[], venue: Venue) => {
   return (
-    partygoers?.filter((partygoer) => partygoer.lastSeenIn[venue.name]) ?? []
+    partygoers?.filter((partygoer) => partygoer.lastSeenIn?.[venue.name]) ?? []
```

Comment:
If `partygoers` are `undefined` this will return `undefined`, then [`currentVenuePartygoers`](https://github.com/sparkletown/sparkle/pull/1042/files#diff-cf26de90c413ef78e360b6f9947fa234a36d61815ce5d3513718f96fa9c06378R29) will return `undefined`, and [`numberOfPartygoers`](https://github.com/sparkletown/sparkle/pull/1042/files#diff-cf26de90c413ef78e360b6f9947fa234a36d61815ce5d3513718f96fa9c06378R33) is going to fail trying to get `.length`

Not sure if the type is catching this, but looks like a possible case.

Reply:
I think the optional parameter here is unnecessary and mislead me to the statement above.

Reply:
I think if partygoers are `undefined` then ?? will work and it will return an empty array

Reply:
Try `partygoers.filter`

Reply:
Sure, it is not finished yet. There are a lot of places, where `??` is used in undesirable places. I think I will fix that in my next commits.